restalchemy/api/controllers.py

```diff
@@ -154,11 +177,24 @@ def do_collection(self, parent_resource=None):
         api_context = self._req.api_context
         if method == "GET":
             api_context.set_active_method(constants.FILTER)
-            filters = self._prepare_filters(
+            order_by = self._prepare_sorts(
                 params=self._req.api_context.params,
             )
+            filters = self._prepare_filters(
+                params=self._req.api_context.params_filters,
+            )
             kwargs = self._make_kwargs(parent_resource, filters=filters)
-            return self.process_result(result=self.filter(**kwargs))
+            try:
+                return self.process_result(
+                    result=self.filter(**kwargs, order_by=order_by)
+                )
+            except TypeError:
+                LOG.warning(
```

Comment:
@phantomii we may just bump major version and remove this warning instantly, but I'm ok either way.

Reply:
Dropped it, will bump major version.